.travis.yml

```diff
@@ -1,23 +1,49 @@
 language: python
 
-dist: xenial
-
 matrix:
-    include:
-      - python: 2.7
-        env: TOXENV=py27
-      - python: 3.5
-        env: TOXENV=py35
-      - python: 3.6
-        env: TOXENV=py36
-      - python: 3.7
-        env: TOXENV=py37
-      - python: 3.7
-        env: TOXENV=pep8
+  include:
+    - os: linux
+      dist: xenial
+      python: 2.7
+      env: TOXENV=py27
+    - os: linux
+      dist: xenial
+      python: 3.5
+      env: TOXENV=py35
+    - os: linux
+      dist: xenial
+      python: 3.6
+      env: TOXENV=py36
+    - os: linux
+      dist: xenial
+      python: 3.7
+      env: TOXENV=py37
+    - os: linux
+      dist: xenial
+      python: 3.7
+      env: TOXENV=pep8
+    - os: linux
+      dist: bionic
+      python: 2.7
+      env: TOXENV=py27
+    - os: linux
+      dist: bionic
+      python: 3.6
+      env: TOXENV=py36
+    - os: linux
+      dist: bionic
+      python: 3.7
+      env: TOXENV=py37
+    - os: linux
+      dist: bionic
+      python: 3.7
+      env: TOXENV=pep8
```

Comment:
I don't think we need to run `pep8` twice on both `dist`. I propose we *only* run this `bionic` env:

```yaml
    - os: linux
      dist: bionic
      python: 3.7
      env: TOXENV=py37
```

We don't need to test 2.7, 3.6, and pep8 again on bionic. And it simplifies the number of tests and should make the whole test process run faster.